frontend/packages/component-library/src/image/Image.tsx

```diff
@@ -46,31 +48,36 @@ const Image: React.FC<ImageProps> = ({
   className,
   onLoad,
   onError,
+  imageComponent,
   ...ImageHTMLAttributes
-}) => (
-  <figure
-    className={classNames(
-      moduleStyles.figureContainer,
-      {
-        [moduleStyles['figure-hasBorder']]: decoration === 'border',
-        [moduleStyles['figure-hasBoxShadow']]: decoration === 'shadow',
-        [moduleStyles['figure-hasRoundedCorners']]: hasRoundedCorners,
-      },
-      className,
-    )}
-    // Only use inline styles if there's no way to add custom className with needed styles.
-    style={style}
-  >
-    <img
-      className={classNames(moduleStyles.image)}
-      alt={altText}
-      loading={loading}
-      src={src}
-      onLoad={onLoad}
-      onError={onError}
-      {...ImageHTMLAttributes}
-    />
-  </figure>
-);
+}) => {
+  const ImageComponent = imageComponent ? imageComponent : 'img';
```

Comment:
I'm unable to add gh code suggrstion here, but if we'll set the 'img' default value where I'm suggesting - we won't need this line